### ISSUE TITLE
CORDOVA_COMPATIBILITY_VERSION_EXCLUDE not working

### ISSUE THREAD

**Issue by lorensr** [meteor/meteor#8814]
New Meteor app with platform `ios` and one native plugin, `cordova-plugin-camera`:

https://github.com/lorensr/compat-repro

The `ios` `cordovaCompatilityVersion` is the same for the two below commands (`372fa3b07a8171585fb0b5a26763edd4c5a54feb`). I believe they should be different.

```
meteor run ios-device
CORDOVA_COMPATIBILITY_VERSION_EXCLUDE='cordova-plugin-camera' meteor run ios-device
```

Feature implemented in #8581 by @wojtkowiak

**Comment by wojtkowiak** [comment]
Hey,

the commit message is misleading because that was my proposal and during the review the naming was changed. The correct is `METEOR_CORDOVA_COMPAT_VERSION_EXCLUDE`. Please try that.

**Comment by lorensr** [comment]
Thanks, that worked! New version is `3ed5b9318b2916b595f7721759ead4d708dfbd46`.

Documented in https://github.com/meteor/guide/commit/8866b9f35f1f8ba66f25cc81981f2b4238df59e5

**Comment by lorensr** [comment]
@wojtkowiak Looks like `__meteor_runtime_config__.autoupdateVersionCordova` in JS matches with `manifest.json: version`, but do you know if `manifest.json: cordovaCompatilityVersion.ios` is accessible in JS?

**Comment by wojtkowiak** [comment]
Hmm I would say no. Even the webapp cordova plugin gets it from the manifest instead of receiving that from js side. 
https://github.com/meteor/cordova-plugin-meteor-webapp/blob/master/src/android/AssetBundle.java#L89
You should not have any problems in reading the current manifest from file directly with `cordova-plugin-file`.

**Comment by AndyNormore** [comment]
I'm still having all sorts of issues with this HCP not working.

I have globals set on the web server, windows machine that compiles the web server too.

I have globals set on my mac that publishes android and iOS apps.

None of it works.

**Comment by AndyNormore** [comment]
How do I check what Cordova app my website is running? www.StarCommanderOnline.com

**Comment by wojtkowiak** [comment]
http://www.starcommanderonline.com/__cordova/manifest.json
![obraz](https://user-images.githubusercontent.com/8668306/30017361-2d60b6b0-9159-11e7-9bba-833b4bf25258.png)

**Comment by AndyNormore** [comment]
Solved the problem.

As I wrote in my previous post, I was running on TWO MACHINES. Mac did iOS and Android, while my windows machine built the server.

Once I started uploading the web package from the Mac, it all worked!

tldr; build on one machine.
